scrapers/src/extractors/office.rs

```diff
@@ -0,0 +1,321 @@
+use std::sync::OnceLock;
+
+use regex::Regex;
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct OfficeMeta {
+    pub name: String,
+    pub title: String,
+    pub r#type: Option<String>, // TODO - What determines this?
```

Comment:
I'm unsure what this is?

Reply:
ah haha sorry, yeah this needs a little more context. This `type` field is intended to map to the `office_type` column on the `office` table. Looking at existing data in the database, I wasn't seeing a pattern for what possible values could be used to populate this field.

Reply:
we can omit office_type for now, that was a carryover field from votesmart that we copied and only used for a few offices.  i'll make a note in linear that we should consider deprecating this field